src/java/org/apache/cassandra/tcm/serialization/Version.java

```diff
@@ -36,13 +39,24 @@ public enum Version
     /**
      *  - Added version to PlacementForRange serializer
      *  - Serialize MemtableParams when serializing TableParams
+     */
+    V2(2),
+
+    /**
      *  - Added AccordFastPath
+     *  - Added ConsensusMigrationState
      *  - Added AccordStaleReplicas
+     *  - TableParam now has pendingDrop (accord table drop is multistep)
      */
-    V2(2),
+    V3(3),
```

Comment:
this now conflicts with https://issues.apache.org/jira/browse/CASSANDRA-19954 on trunk....

Believe Sam/Marcus plan to also add more to V3... so Accord is likely V4, but with the `MIN_ACCORD_VERSION` it should be easier to handle when that happens